taskcluster/fenix_taskgraph/transforms/mark_as_shipped.py

```diff
@@ -19,10 +19,10 @@
 def make_task_description(config, jobs):
     for job in jobs:
         product = "Fenix"
-        version = config.params['release_version'] or "{ver}"
+        version = config.params['version'] or "{ver}"
```

Comment:
I'm guessing something populates `{ver}`? I don't see where that happens https://github.com/mozilla-mobile/fenix/search?q=ver&unscoped_q=ver . Or is that just a placeholder for taskgraph full when we're not run through an action?

Reply:
The latter. I had the same question when it was first implemented :) https://github.com/mozilla-mobile/fenix/pull/7306#discussion_r360248631